target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: MIT, GPL-2.0 or later
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+
+#include "ipq6018.dtsi"
+#include "ipq6018-cp-cpu.dtsi"
+#include "ipq6018-ess.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Yuncore FAP650";
+	compatible = "yuncore,fap650", "qcom,ipq6018-cp03", "qcom,ipq6018";
+	
+	/delete-node/ rpm-glink;
```

Comment:
No, dont delete stuff but rather just disable it.

Reply:
Previously there was a problem with rebooting the device without this instruction.

Just removed it now.